vms_design_document.md

```diff
@@ -0,0 +1,8 @@
+# Violation Message Suppression (VMS) Implementation
```

Comment:
Why have a second design document?

Reply:
This was a holdover from when we were first thinking about vms, it's been removed now